scripts/sync-kernel.sh

```diff
@@ -357,5 +357,15 @@ cd_to "${LINUX_REPO}"
 git checkout "${TIP_SYM_REF}"
 cd_to "${LIBBPF_REPO}"
 "${LIBBPF_REPO}"/scripts/mailmap-update.sh "${LIBBPF_REPO}" "${LINUX_REPO}"
+# if anything changed, commit it
```

Comment:
looks good, but let's move entire .mailmap step to right after bpf_helpers_defs.h, so that "cover letter" commit includes it as well in the summary?

Reply:
Sounds good, I moved it. The new commit will be “before” the cover letter, but will not be listed in the summary, which is generated before that.

Reply:
Ah, ok, seems like we don't put bpf_helper_defs.h into cover letter as well, that's fine. But I think this order of commits is more logical, thanks!